sal_interfaces/SALSubsystems.xml

```diff
@@ -759,11 +759,11 @@
   <CSCDocs></CSCDocs>
```

Comment:
GH review won't let me put this where I want. HVAC JenkinsTestResults cannot be empty.

Reply:
I changed it to "Not Available".